.gitignore

```diff
@@ -40,3 +40,6 @@ testem.log
 #System Files
 .DS_Store
 Thumbs.db
+
+*node_modules
+src/angularjs/dist
```

Comment:
Has little to do this with line, but I think that we should maintain the existing directory structure vs. tucking all of the Angular stuff into `src/angularjs`. It appears as though the directory structure that exists in the repository now is created by the Angular CLI.